loop_extract.cpp

```diff
@@ -253,7 +254,13 @@ int main(int argc, char **argv) {
   snprintf(attribute_qsq,100,"%d",(int)g_filtered_qsq);
 
 
-  snprintf ( filename, 400, "filtered_%s.%.4d_%s_Ns%.4d_step%.4d_Qsq%d_gamma%d.h5", filename_prefix, Nconf, filename_prefix2, g_nsample, Nsave, (int)g_filtered_qsq, g_currentgammas );
+  if (g_currentgammas != 4){
```

Comment:
Could you just add an option to actually disable the spin projection? I would prefer if no transformation were applied to the data (even if it is just the identity)